content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,293 @@
+
+</WarningBox>
+
+- Minimum capacity requirements as follows:
+
+    <br />
+
+    - One node (no HA): 4 vCPU, 8 GiB Memory, 60 GiB Storage.
+   
+    - Three nodes (HA): 6 vCPU, 12 GiB Memory, 90 GiB Storage.<br /><br />
+
+- An active MAAS API key in the following format:
+
+  ``[consumer_key], [key], and [secret] tokens: API key = '[consumer_key]:[key]:[secret]'``
+
+  <br />
+
+- The DNS server that the installer will use must be able to resolve the public internet names of the machines that MAAS manages so it can connect to them.  
+
+    This is commonly done by ensuring the DNS server delegates the MAAS domain to the MAAS control plane. To connect, the installer uses the FQDN ``machine-hostname.maas``. A common way to enable this is to ensure the DNS server delegates the MAAS domain to the MAAS control plane.
+
+# Understand the Gateway Installation Process
+
+The following steps outline the overall process to install the PCG. For detailed steps, refer to [Install the Gateway](/clusters/data-center/maas/install-manage-maas-pcg#installthegateway) below.
+
+<br />
+
+1. You obtain a pairing code in Palette that you will use later.
+
+
+2. Then you run a configuration script using a docker image. 
+
+
+3. The script creates a PCG configuration file.
+
+
+4. Next, you run the installer using the open-source Kind tool and provide the pairing code you obtained in step **1**. The installer uses the configuration file from script output in step **3**.
+
+    The installer needs access to your Palette account and to one or three machines in your MAAS cluster. If you select one machine in Palette, then you need one in MAAS. Likewise, if you select three machines in Palette, you need three in MAAS. The MAAS machines must have internet access and be in a ready state.
+    <br />
+
+5. The installer installs to the MAAS machine(s) and uses the configuration file to build a new cluster to host the PCG application. 
+
+
+
+# Install the Gateway
+
+The following steps will guide you to install the PCG. 
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main** menu and select **Tenant Settings > Private Cloud Gateway**.
+
+
+3. Click the **Create Private Cloud Gateway** button and select **MAAS**. Private Gateway installation instructions are displayed.
+
+
+4. Copy the pairing code displayed in the Instructions section of the page and paste it in a text file. You will input this code when you run the installer. 
+
+
+5. Copy the following code snippet to your terminal to invoke the installer.
+    <br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.12 \
+    -o true
```

Comment:
```suggestion
    docker run -it --rm \
    --net=host \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    --volume /tmp:/opt/spectrocloud \
    gcr.io/spectro-images-public/release/spectro-installer:1.0.12
```
Adding the longform of the commands. --rm and --it are best left as it. Also removed the `-o` flag.